game/scripts/npc/abilities/visage_grave_chill.txt

```diff
@@ -48,7 +48,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "attackspeed_bonus"                               "34 46 58 70 82 94"
+        "attackspeed_bonus"                               "36 48 60 72 96 120" //OAA
```

Comment:
You can put similar values to Shiva's Guard. Keep in mind that bonus AS affects familiars too.

Reply:
shivas values seems a little too good, Do you dislike these current numbers

Reply:
I don't dislike them, it just doesn't seem impactful. It doesn't seem like it will change something about the hero.

Reply:
Purpose of these changes are more or less more impactful at oaa levels to counteract shivas a bit better, early levels is just to look prettier and scale correctly with oaa levels.